docs/embedding/interactive-ui-components.md

```diff
@@ -89,35 +111,27 @@ If only tables are allowed, the sidebar won't show models:
 
 ![Sidebar without models](./images/sidebar-without-models.png)
 
-The new button dropdown (if `new_button=true`) will also exclude models:
-
-![New button](./images/new-button-dropdown.png)
-
 Available entity types are:
 
 - `table`
 - `model`
+- `question`
```

Comment:
`question` only works with `data_picker=staged`, so we should call it out probably.